doc/example/patterns_static_objects.cpp

```diff
@@ -6,18 +6,19 @@
 // (See accompanying file LICENSE_1_0.txt or copy at
 // http://www.boost.org/LICENSE_1_0.txt)
 
+// Used to make this test file pass. Define to empty to see other tests fail
+#define ASSERT_VERIFY_FAIL() mock::reset()
+
 //[ static_objects_problem
-#define BOOST_AUTO_TEST_MAIN
-#include <boost/test/auto_unit_test.hpp>
+#include <boost/test/unit_test.hpp>
 #include <turtle/mock.hpp>
 #include <ostream>
 
 namespace
 {
     struct my_class
     {
-        my_class( int i )
-            : i_( i )
+        my_class( int i ) : i_( i )
```

Comment:
OK, we can change the code formatting, I don’t really care which to choose, but this needs to be consistent everywhere 😁